File: pepco-scraper/src/main/scala/com/pocketcookies/pepco/scraper/PointDouble.scala
package com.pocketcookies.pepco.scraper

class PointDouble(val lat: Double, val lon: Double) {
  def this(sLatLon: List[String]) = this(java.lang.Double.parseDouble(sLatLon(0)), java.lang.Double.parseDouble(sLatLon(1)))
}

File: pepco-scraper/src/main/java/com/pocketcookies/pepco/scraper/PointDouble.java
package com.pocketcookies.pepco.scraper;

public class PointDouble {
    public final double lat;
    public final double lon;

    public PointDouble(final double lat, final double lon) {
        this.lat = lat;
        this.lon = lon;
    }

    public PointDouble(String latLng) {
        this(latLng.split(" "));
    }

    private PointDouble(String[] latLng) {
        this(Double.parseDouble(latLng[0]), Double.parseDouble(latLng[1]));
        assert latLng.length == 2;
    }
}
